src-go/server/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.20
 
 require (
 	github.com/ooni/oohttp v0.6.1
+	github.com/pkg/errors v0.9.1
```

Comment:
This package is [deprecated](https://github.com/pkg/errors) and probably unneeded. Please remove if possible.

Reply:
fixed